packages/ui/components/pages/Fuse/Modals/PoolModal/AmountSelect.tsx

```diff
@@ -287,6 +293,11 @@ const AmountSelect = ({
     }
   }, [mode, updateAvailableToWithdraw]);
 
+  const token = getContract(asset.underlyingToken, ERC20Abi, midasSdk.signer);
```

Comment:
Can't see the whole context here. Could maybe better be in a useEffect() hook or useMemo.

Otherwise it will be executed on each render, which is not good at all.

Reply:
this line is now in a useEffect hook